content/partials/import-mount-functions.md

```diff
@@ -8,7 +8,7 @@ framework-specific `mount()` functions, which can be imported like so:
 <strong class="alert-header">A note for React users</strong>
 
 The `mount()` command exported from the
-[@cypress/react](https://github.com/cypress-io/cypress/tree/develop/npm/react)
```

Comment:
I'm not sure we want to call `cypress/react` a library. `@cypress/react` is a library, and `cypress` is a library, but `cypress/react` is really the `react` path under the `cypress` library.